src/adv.py

```diff
@@ -33,19 +44,180 @@
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
 
+# Load room with items
+room['outside'].items = ['stick', 'lantern oil']
+room['foyer'].items = ['leather boots']
+room['overlook'].items = ['telescope', 'binoculars']
+room['narrow'].items = ['painting']
+room['treasure'].items = ['dry blood']
+
 #
 # Main
 #
 
+# Start in outside room with an empty rucksack
+location = 'outside'
+items = ['rucksack']
+
 # Make a new player object that is currently in the 'outside' room.
 
+class PlayerOne(Player):
+    def __init__(self, name, condition, location, items):
+        self.name = name
+        self.condition = condition
+        self.location = location
+        self.items = items
+        
+player_info = PlayerOne('Dylan', 'healthy', location, items)
+print(player_info.items)
+
+SCREEN_WIDTH = 50
+
 # Write a loop that:
-#
 # * Prints the current room name
 # * Prints the current description (the textwrap module might be useful here).
 # * Waits for user input and decides what to do.
-#
 # If the user enters a cardinal direction, attempt to move to the room there.
 # Print an error message if the movement isn't allowed.
-#
 # If the user enters "q", quit the game.
+
+def moveDirection(direction):
+    """A helper function that changes the location of the player."""
+    if direction == 'north' and hasattr(room[player_info.location], 'n_to') and (room[player_info.location].n_to != room[player_info.location]):        
+        if hasattr(room[player_info.location], 'n_to'):
+            # cur_rm_desc = room[player_info.location]
+            to_rm_desc = room[player_info.location].n_to
+            for i in room.keys():                
+                if player_info.location == i and hasattr(room[player_info.location], 'n_to'):                    
+                    for i in room.keys():
+                        if to_rm_desc == room[i]:
+                            player_info.location = i
+        print('Current room: ', player_info.location)
+        print(f'You are currently in {player_info.location}:\n\nRoom description:\n{room[player_info.location]}\n\n')
+    elif direction == 'south' and hasattr(room[player_info.location], 's_to') and (room[player_info.location].s_to != room[player_info.location]):        
```

Comment:
A lot of your codes for south, east, and west moves can be reusing codes for north move.  Try to make this dryer when you get a chance